src/source/Signaling/Signaling.c

```diff
@@ -863,10 +862,6 @@ STATUS describeChannel(PSignalingClient pSignalingClient, UINT64 time)
 
 CleanUp:
 
-    if (STATUS_FAILED(retStatus) && pSignalingClient != NULL) {
```

Comment:
Not sure what the reason is for removing these checks

Reply:
It is very unclear to me why on failed status we would override the result value to UKKNOWN.  The call result has useful information regarding in what way the call failed.

Reply:
I can't recall the reason for this or where this is checked or what edge case this is used. I have not followed all the recent changes so perhaps this is a no-op impact right now. I would only suggest to check whether this is somehow used in retriable failure cases but those should be covered in the unit tests so if they pass then I think its great that this gets removed.